test-e2e/tests/watch_history/logged_in_test.ts

```diff
@@ -84,7 +63,7 @@ function runTestSuite(config: typeof testConfigs.svod, configNoWatchlist: typeof
   Scenario(`I do not see continue_watching videos on the home page and video page if there is not such config setting - ${providerName}`, async ({ I }) => {
     I.useConfig(configNoWatchlist);
 
-    await registerOrLogin(I);
+    I.registerOrLogin();
```

Comment:
Don't you need an await here?

Reply:
I believe going off this part of the documentation that I.* calls don't necessarily need to be awaited (https://codecept.io/basics/#waiting) but correct me if I misunderstood

Reply:
I'm pretty sure this would only apply if your entire logic flow was a series of `I.` calls, which ours is not.

"If you want to get information from a running test you can use await inside the async function, and utilize special methods of helpers started with the grab prefix."

If you dig down into the methods called inside `registerOrLogin`, you'll eventually find at least 1 `grab` which I think means dropping the await is going to cause problems.

Looks like you can probably remove these 2 awaits:
- https://github.com/jwplayer/ott-web-app/blob/develop/test-e2e/utils/steps_file.ts#L76 - kind of a silly check to worry about. I would say just delete the line
- https://github.com/jwplayer/ott-web-app/blob/develop/test-e2e/utils/steps_file.ts#L291 - this one I'm unsure of exactly how it works properly. I think the usePlaywrightTo method used to allow a return value but no longer does. It would be really nice to find a way to replace this with an injected param instead of checking it each time

Then I think a lot fewer tests will need to use async and await, but when you see it, it's probably a good idea to wait for it.